test/testbeast.cpp

```diff
@@ -39,28 +39,41 @@ int main() {
         return "";
     });
 
-    // Listen on default port.
-    assert(std::system("lsof -i:32489") == 0);
+    // Listen on default unix socket.
+    assert(std::system("lsof /tmp/fcitx5.sock") == 0);
 
     // Getter works.
     assert(!getterCalled);
-    assert(std::system("curl http://localhost:32489/config/addon/beast") == 0);
+    assert(std::system("curl --unix-socket /tmp/fcitx5.sock "
+                       "http://fcitx/config/addon/beast") == 0);
     assert(getterCalled);
 
     // Setter works.
     assert(!setterCalled);
-    assert(
-        std::system(
-            "curl -X POST -d '{}' http://localhost:32489/config/addon/beast") ==
-        0);
+    assert(std::system("curl --unix-socket /tmp/fcitx5.sock -X POST -d '{}' "
+                       "http://fcitx/config/addon/beast") == 0);
```

Comment:
This removes TCP test for getter/setter called. Please add them back.

Reply:
Done via d047c61